docs/src/main/sphinx/release/release-399.md

```diff
@@ -0,0 +1,74 @@
+# Release 399 (5 Oct 2022)
+
+## General
+
+* Add operator CPU and wall time distribution to `EXPLAIN ANALYZE VERBOSE`. ({issue}`14370`)
+* Improve performance of joins. ({issue}`13352`)
+* Remove support for the deprecated `row` to `json` cast behavior and remove the
+  `deprecated.legacy-row-to-json-cast` configuration property. ({issue}`14388`)
+* Fix error when using `PREPARE` with `DROP VIEW` when the view name is quoted. ({issue}`14196`)
+
+## Accumulo connector
+
+* Change the default value of the `accumulo.zookeeper.metadata.root`
+  configuration property to `/trino-accumulo` from `/presto-accumulo`. ({issue}`14326`)
+
+## BigQuery connector
+
+* Add support for writing `array` and `row` columns. ({issue}`14418`)
+
+## ClickHouse connector
+
+* Fix bug where the intended default value of the `domain-compaction-threshold`
+  configuration property was instead a maximum limit. ({issue}`14350`)
+
+## Delta Lake connector
+
+* Improve performance of reading decimal columns from Parquet files. ({issue}`14260`)
+* Allow setting the AWS STS endpoint and region when using a Glue metastore. ({issue}`14412`)
+
+## Hive connector
+
+* Add `max-partition-drops-per-query` configuration property to limit the number
+  of partition drops. ({issue}`12386`)
+* Add `hive.s3.region` configuration property to force S3 to connect to a
+  specific region. ({issue}`14398`)
+* Improve performance of reading decimal columns from Parquet files. ({issue}`14260`)
+* Reduce memory usage of the coordinator. ({issue}`14408`)
```

Comment:
Why is this in the Hive connector section? Also  .. memory usage on the coordinator?

Reply:
It only impacts Hive, it has to do with caching Hadoop LocatedFileStatus objects.